lib/purgatory/purgatory.rb

```diff
@@ -44,9 +44,14 @@ def soul_with_changes
   end
 
   def approve!(approver = nil)
-    return false if approved? 
-    if soul_with_changes.save
-      soul.send(performable_method[:method],*performable_method[:args]) if performable_method
+    return false if approved?
+    success = nil
```

Comment:
@dobsonken did you see the other suggestion I made here at the top? Probably should have made it two separate comments. Sorry about that. But yeah I think it would be better with the style I suggested in the code snippet above.